esmvaltool/cmor/_fixes/CMIP5/HadGEM2_CC.py

```diff
@@ -21,9 +21,11 @@ def fix_metadata(self, cube):
         iris.cube.Cube
 
         """
-        lat = cube.coord('latitude')
-        lat.points = np.clip(lat.points, -90., 90.)
-        lat.bounds = np.clip(lat.bounds, -90., 90.)
+        lats = cube.coords('latitude')
+        if len(lats):
```

Comment:
```suggestion
        if lats:
```
as Codacy suggests, an empty `list` evaluates to `False`

Reply:
Also reproduced this solution in the HadGEM2_ES fixes script.